core/designsystem/src/main/kotlin/com/google/samples/apps/nowinandroid/core/designsystem/component/scrollbar/ScrollbarExt.kt

```diff
@@ -32,42 +33,57 @@ import kotlinx.coroutines.flow.filterNotNull
 import kotlin.math.min
 
 /**
- * Calculates a [ScrollbarState] driven by the changes in a [LazyListState].
+ * Calculates a [ScrollbarState] driven by the changes in a [LazyState].
  *
- * @param itemsAvailable the total amount of items available to scroll in the lazy list.
- * @param itemIndex a lookup function for index of an item in the list relative to [itemsAvailable].
+ * @param itemsAvailable the total amount of items available to scroll in the [LazyState].
+ * @param itemSize a lookup function for the size of an item in the layout.
+ * @param offset a lookup function for the offset of an item relative to the start of the view port.
+ * @param nextItemOnMainAxis a lookup function for the next item on the main axis in the direction
+ *  of the scroll.
+ * @param itemSize next [LazyStateItem] on the main axis of [LazyState], null if none.
+ * @param index a lookup function for index of an item in the layout relative to
+ * @param viewportStartOffset a lookup function for the start offset of the view port
+ * @param viewportEndOffset a lookup function for the end offset of the view port
+ * @param reverseLayout a lookup function for the reverseLayout of [LazyState].
  */
 @Composable
-fun LazyListState.scrollbarState(
+internal inline fun <LazyState : ScrollableState, LazyStateItem> LazyState.genericScrollbarState(
```

Comment:
Is worth to make this function inline? inline keyword isn't recommended for huge functions, since it can make the bytecode to grow

Reply:
Sure. I agree, I removed the inline qualifier, and also made it more restrict the access to this function, from internal to private, since this function is only used in that file.
Thanks for the review @machado001.